EjercicioDiaTres.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciosProyect_Visual.Net
+{
+    class EjercicioDiaTres
+    {
+        static EjercicioDiaTres ejercicioDia;
+        public static void Main(string[] args)
+        {
+            ejercicioDia = new EjercicioDiaTres();
+            ejercicioDia.llamarEjercicioUno();
+            ejercicioDia.llamarEjercicioDos();
+            ejercicioDia.llamarEjercicioTres();
+        }
+        public void llamarEjercicioUno()
+        {
+            int resultado;
+            Console.WriteLine("Ingresa el Primer número: ");
+            int numero1 = int.Parse(Console.ReadLine());
```

Comment:
Recuerda el nombramiento de este tipo de variables.
```suggestion
            int numeroUno = int.Parse(Console.ReadLine());
```

Reply:
Comprendido y Realizado